src/preferences/dialog/dlgprefcontrolsdlg.ui

```diff
@@ -489,6 +489,23 @@
        </property>
       </widget>
      </item>
+     <item row="12" column="1" colspan="2">
+      <widget class="QCheckBox" name="checkBoxDoubleWidgetSize">
```

Comment:
The desired scale factor may vary depending on screen resolution, physical screen size, and personal taste, so I think it would be good to make it more flexible.